test/cmd/newapp.sh

```diff
@@ -11,13 +11,12 @@ os::log::install_errexit
 
 # This test validates the new-app command
 
-os::cmd::expect_success 'oc create -f examples/image-streams/image-streams-centos7.json'
-
 os::cmd::expect_success_and_text 'oc new-app library/php mysql -o yaml' '3306'
 os::cmd::expect_failure 'oc new-app unknownhubimage -o yaml'
 # verify we can generate a Docker image based component "mongodb" directly
 os::cmd::expect_success_and_text 'oc new-app mongo -o yaml' 'library/mongo'
 # the local image repository takes precedence over the Docker Hub "mysql" image
```

Comment:
is just checking for the text `mysql` in `os::cmd::expect_success_and_text 'oc new-app mysql -o yaml' 'mysql'` really testing precedence? I would expect it to contain `mysql` even if it was referencing dockerhub